README.md

```diff
@@ -59,6 +59,13 @@ PasswordAuthentication no
 UsePAM no
 ```
 
+For users that⁠—for [security reasons](https://yunohost.org/en/security#ssh-authentication-via-key)⁠—disabled the admin password and now perform SSH authentication via key, keep `UsePAM yes` and add `AllowUsers gitea`:
+
+```bash
+UsePAM yes
+AllowUsers gitea
```

Comment:
I'm pretty sure that indeed th the part about `UsePAM no` in the current readme is fucked up and breaks many things, it should be removed and probably has nothing to do with gitea

`AllowUsers gitea` is probably not a good idea either, the right way should be to `sudo adduser gitea ssh.app`, then gitea should be allowed because of the ~recent `AllowGroups ssh.app` line that got added to the sshd_config in Yunohost 4.1 or 4.2